README.md

```diff
@@ -279,6 +279,55 @@ Example for sending data to EU site:
     datadog_api_key: "123456"
 ```
 
+### Making the playbook work on Windows
+
+On Windows, the `become: yes` option only works if:
+- `become_method` is set to `runas`,
+- `become_user` is set to a user with Administrator powers (for instance, `System`).
+
+Below are two methods to make the above playbook work with Windows hosts:
+
+### Using the inventory file (recommended)
+
+The recommended way to set these options is to set then in the inventory file, by setting the `ansible_become_method` and `ansible_become_user` options:
+
+```ini
+[servers]
+linux1 ansible_host=127.0.0.1
+linux2 ansible_host=127.0.0.2
+windows1 ansible_host=127.0.0.3 ansible_become_method=runas ansible_become_user=System
```

Comment:
`ansible_become_user` is optional I think. By default it will use the `Winrm` user, no ?